frontend-react/src/pages/TermsOfService.tsx

```diff
@@ -221,44 +222,41 @@ export const TermsOfService = () => {
 
                         <p>
                             PRIME ReportStream is being provided to allow for
-                            the transmission of Public Health Data to STLT
-                            Public Health Agencies to facilitate authorized
-                            public health activities. HHS and CDC acknowledge
-                            that CDC, through PRIME ReportStream, is acting as a
-                            conduit for transmission of Public Health Data to
-                            STLT Public Health Agencies for the purposes of
-                            public health reporting and does not intend to take
-                            custody or control of those data. Where the Public
-                            Health Data is not in CDC's custody and control and
-                            the Public Health Data is requested of CDC, CDC
-                            shall notify the requestor that CDC does not have
-                            custody of this data and refer requestor to the
-                            Entity.
+                            the transmission of Public Health Data to facilitate
+                            authorized public health activities. As agreed upon
+                            with the relevant User(s), transmissions may be
+                            directly to STLTs, HHS, CDC or to third parties.
+                            Users sending Public Health Data to these entities
+                            directly through PRIME ReportStream agree that any
+                            such transmission is consistent with applicable law
+                            and any applicable agreements.
+                        </p>
+
+                        <p>
+                            Where HHS or CDC are direct recipients as agreed
+                            upon with the relevant User(s), such Public Health
+                            Data in HHS’s or CDC’s custody and control will be
```

Comment:
This is the SMALLEST, STUPIDEST POINT IN THE WORLD, but let's use actual single quotes rather than smart quotes: 

```jsx
HHS&apos;s or CDC&apos;s
```

The reason for this is that they have separate character codes so they can map to different glyphs in certain typefaces, and single quotes are part of a standard set of assumed glyphs for any typeface.  Really not a big deal because there are graceful fallbacks when there are missing glyphs, though.

Again, SMALLEST, STUPIDEST POINT IN THE WORLD 😆

Reply:
Fixed a few I found: https://github.com/CDCgov/prime-reportstream/pull/8171/commits/3cf40fd6f7be053f8f026f765dbab8987347f44a